sdk/cosmos/azure-cosmos-benchmark/src/main/java/com/azure/cosmos/benchmark/Configuration.java

```diff
@@ -163,7 +163,8 @@ public Duration convert(String value) {
         Mixed,
         ReadMyWrites,
         ReadThroughputWithMultipleClients,
-        CtlWorkload;
+        CtlWorkload,
+        ReadAllItemsOfLogicalPartition;
```

Comment:
please add one line description to "-operation" parameter description, similar to other operations.

Reply:
Fixed